### PR TITLE
Add planning-scheduler page for Vue

### PR DESCRIPTION
**Pull request by GoodDayForSurf**

### CODE REVIEW

**Comment by VasilyStrelyaev** [comment]
With this PR, the navigation sidebar now has the same items for all platforms.
It means that we should remove a lot of masks that were introduced to suppress test failures because of the `Scheduler` item was present in React but absent from Angular and Vue.

File: packages/vue/src/pages/planning-scheduler.vue [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+          <dx-speed-dial-action
+            icon="add"
+            :visible="screenInfo.isXSmall"
+            @click="showAppointmentCreationForm"
+          />
+        </div>
+
+        <right-side-panel
+          class="right-side-panel"
+          :is-opened="isRightPanelOpen"
```
Comment (VasilyStrelyaev) [inline packages/vue/src/pages/planning-scheduler.vue:78]: If the right panel is opened when the Month view is selected, should it be closed automatically if the view is changed back?

File: packages/vue/src/components/utils/agenda-list-item.vue [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  resources: Record<string, unknown>[],
+}>(), {
+  items: () => ({}),
+  resources: () => [],
+});
+
+const startTime = computed(() => props.appointment.startDate.toLocaleTimeString(undefined, {
+  hour: 'numeric',
+  minute: 'numeric',
+  hour12: false,
```
Comment (VasilyStrelyaev) [inline packages/vue/src/components/utils/agenda-list-item.vue:37]: The scheduler shows a 12-hour time, while the agenda view and tooltip show a 24-hour time.

Let's be consistent, choose a format and use it everywhere?

File: packages/vue/src/pages/planning-scheduler.vue [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+  agendaItems.value = findAllAppointmentsForDay(appointmentData);
+}
+
+function onCalendarDateChange({ value }: { value: Date }) {
+  currentDate.value = value;
+  updateAgenda({ startDate: currentDate.value });
+}
+
+function showAppointmentCreationForm(appointment: SelectedAppointment| null) {
+  schedulerRef.value?.instance.showAppointmentPopup(appointment?.data, !appointment);
```
Comment (VasilyStrelyaev) [inline packages/vue/src/pages/planning-scheduler.vue:203]: Should this form actually create an appointment? It doesn't create one for me...

File: packages/vue/src/pages/planning-scheduler.vue (hunk 1 above)
Comment (GoodDayForSurf) [reply packages/vue/src/pages/planning-scheduler.vue:78]: it's approved by designer

File: packages/vue/src/pages/planning-scheduler.vue (hunk 3 above)
Comment (GoodDayForSurf) [reply packages/vue/src/pages/planning-scheduler.vue:205]: fixed

File: packages/vue/src/components/utils/agenda-list-item.vue (hunk 2 above)
Comment (GoodDayForSurf) [reply packages/vue/src/components/utils/agenda-list-item.vue:37]: looks like it need to discuses for next step improvements with designer

File: packages/vue/src/components/utils/calendar-list.vue [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    :hover-state-enabled="false"
+    :grouped="true"
+    :collapsible-groups="true"
+    :scrolling-enabled="false"
+  >
+    <template #group="{ data: item }">
+      <div class="list-header">
+        {{ item.key }}
+        <dx-button
+          icon="add"
```
[the remark this answers is not in the document]
Comment (GoodDayForSurf) [reply packages/vue/src/components/utils/calendar-list.vue:16]: works as designed

**Review by ivanblinov2k17 — APPROVED** [review]

**Review by VasilyStrelyaev — APPROVED** [review]

**Pull request merged**